store/postgres/asset_repository.go

```diff
@@ -634,6 +634,7 @@ func (r *AssetRepository) getAssetVersionSQL() sq.SelectBuilder {
 		LeftJoin("users u ON a.updated_by = u.id")
 }
 
+// buildFilterQuery retrieves the sql query based on applied filter in the queryString
 func (r *AssetRepository) buildFilterQuery(builder sq.SelectBuilder, cfg asset.Config) sq.SelectBuilder {
```

Comment:
this looks like a complex query builder. 
can we have a test for this?

Reply:
I think every single query builder in `AssetRepository.buildFilterQuery` method is tested in `AssetRepositoryTestSuite.TestGetAll` method. So does it will create any difference on testing it again.
WDYT ??

Reply:
@scortier true but since it has a complex logic, better to be tested again alone. `Given the config, what psql query do we expect`

Reply:
okay, sure will add it 👍🏻